src/rules/arrowParensRule.ts

```diff
@@ -72,8 +72,10 @@ function walk(ctx: Lint.WalkContext<Options>) {
                 }
             } else if (ctx.options.banSingleArgParens) {
                 const closeParen = getChildOfKind(node, ts.SyntaxKind.CloseParenToken)!;
+                const parentNode = node.parent!;
+                const replaceValue = ts.isCallExpression(parentNode) ? "" : " ";
```

Comment:
This should not be limited to CallExpressions. This is also relevant for NewExpression and probably many other.
I propose to remove the blank all the time if there is no alpha character at `openParen.pos - 1`. That includes `async (a) => b` and `export default (a) => b`.
Another rule will probably complain about a missing blank, but that's not the business of this rule.